coordinator/src/routes.rs

```diff
@@ -542,3 +553,55 @@ pub async fn collaborative_revert_confirm(
     })?;
     Ok(Json(serialize_hex(&raw_tx)))
 }
+
+// TODO(holzeis): There is no reason the backup and restore api has to run on the coordinator. On
+// the contrary it would be much more reasonable to have the backup and restore api run separately.
```

Comment:
Definitely. Maybe we should create from the getgo at least a different subdomain/path/port for the backup to be able to move it.

Reply:
As we are already using a reverse proxy, we could simple direct any request to `/api/backup` or `/api/restore` to a different service once we move it. This could be completely transparent to the app.

Reply:
The problem is though that we are hardcoding the IP in the app. If we were to use `backup.10101.finance` instead, we could move it indepdently.

Reply:
> The problem is though that we are hardcoding the IP in the app.

Agreed, but this is unrelated to this change. e.g. we could change the configuration to use the domain instead of the ip. e.g. `api.10101.finance` and then configure the reverse proxy to route any traffic on `api.10101.finance/api/backup` to another service.

One caveat I see in using a dedicated subdomain is that this would introduce another configuration. What benefits do you see in using a dedicated subdomain?

Reply:
It will be hard to roll this out at a later stage or move the backup to a new server without breaking existing apps. 

i.e. 
1. `user_a` runs version 1.6.0 and backs-up his app on 10101.finance
2. we migrate the backup service to backup.10101.finance
3. `user_a` cannot create/restore a backup before upgrading to the new version

I guess breaking changes are OK. Are we handling errors correctly, i.e. what if the backup service is not working (e.g. for an old version because we moved it already)?

Reply:
> we migrate the backup service to backup.10101.finance
> user_a cannot create/restore a backup before upgrading to the new version

I don't see an issue here. We can simply add an additional nginx config that points all the backup/restore requests from user_a to the new service.

But, let's not argue about that. If you feel strongly about it. I can add it as a follow up to this PR.

Reply:
> I don't see an issue here. We can simply add an additional nginx config that points all the backup/restore requests from user_a to the new service.

Can we do this if the new service runs on a different machine/ip?

Reply:
Yes, we would need to configure a redirect then.